src/game/game.h

```diff
@@ -574,6 +579,7 @@ class Game
 		static constexpr int32_t SUNRISE = 360;
 
 		bool isDay = false;
+		bool browseField = false;
```

Comment:
What and why is this for? 😨

Reply:
it's for quickloot all corpses in the same tile

Reply:
But why is it a variable in the Game context? If anything, shouldn't it be within Player class?

Reply:
Or perhaps local to the `playerQuickLoot` / `playerLootAllCorpses` stacks, if possible